pkg/controller/job/job_controller.go

```diff
@@ -951,6 +951,19 @@ func (jm *Controller) deleteActivePods(ctx context.Context, job *batch.Job, pods
 	return successfulDeletes, errorFromChannel(errCh)
 }
 
+func getNonIgnoredFailedPodsCnt(jobCtx *syncJobCtx, failedPods []*v1.Pod) int {
```

Comment:
```suggestion
func nonIgnoredFailedPodsCount(jobCtx *syncJobCtx, failedPods []*v1.Pod) int {
```

`get` generally doesn't add any value.

Reply:
right, cleaned up